.github/workflows/ci.yml

```diff
@@ -35,7 +35,7 @@ jobs:
     strategy:
       matrix:
         # todo put back?
-        node-version: [16.x, 18.16.0, 20.4.0]
+        node-version: [16.x, 18.16.0, 20.6.0]
```

Comment:
```suggestion
        node-version: [18.16.0, 20.x]
```

Reply:
isn't it safer to use specific versions of node (like we do for deps with yarn.lock)

Reply:
I don't know about safer, but if we pin the Node.js version we can't detect wether our tests pass with newer versions.

Reply:
should we then also remove yarn.lock so we can run our tests for newer versions of npm packages?

Reply:
Why would we? It would create discrepancies between each of our dev environment.

Reply:
I just thought that we already have discrepancies between each of our dev envs when it comes to node version, so maybe we could make it consistent and also do it with npm packages.

Reply:
Having a lock file is easy, forcing devs to all use the same version of Node.js is not. I think we should aim for consistency when it makes sense, the current situation is a good tradeoff IMO